gui-apps/grimblast/grimblast-2022.10.13.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+COMMIT="dafb86d6d3adb421e788f0eaeecdf09c497a14e8"
```

Comment:
please move this into the non-9999 part of PV check.

Reply:
done